proposal/color-4-hwb.md

```diff
@@ -0,0 +1,179 @@
+describing color, but `hwb()` stands out as part of the same `sRGB` color
+system that Sass already uses internally.
+
+[color-4]: https://www.w3.org/TR/css-color-4/
+
+## Summary
+
+> This section is non-normative.
+
+This proposal defines  aSassified versions of the `hwb()` color function
+added to [CSS Color Level 4][color-4] – along with relevant inspection and
+adjustment options. This function will only be available inside the `sass:color`
+module to avoid conflicts with the CSS syntax, and will be converted to more
+common color-name, hex, or `rgba()` syntax for output -- following the same
+logic as our current color functions.
+
+- New `color.hwb()` describes colors in the sRGB colorspace using `$hue` (defined
+  identically to the `hsl()` "hue" value), along with `$whiteness`, `$blackness`,
+  and optional `$alpha` transparency.
+- New `color.whiteness()` and `color.blackness()` functions return the respective
+  values of `w` or `b` for a given color.
+- Existing `color.adjust()`, `color.scale()`, and `color.change()` functions will
+  accept additional `$whiteness` and `$blackness` parameters before the final
+  `$alpha` parameter.
+
+### Design Decisions
+
+For backwards-compatibility reasons, both `rgb/a()` and `hsl/a()` are processed
+as Sass function in the global namespace, and output is converted to the format
+with best browser-support. The primary purpose of Sass color-function support
+is our ability to provide those conversions, but the module syntax now allows
+us to namespace new formats, so authors can opt-into the feature. With that
+in mind:
+
+- The new functions **will not** be available on the global namespace.
+- The new functions **will not** accept *special number string* or
+  *special variable string* values that can only be resolved in CSS.
+
+Unknown CSS functions are already passed through implicitly. As browser support
+grows for the native-CSS functions, authors will be able to migrate on their own
+schedule – or use both Sass & CSS functions together to progressively enhance
+their designs.
+
+Since all `color.hwb()` output will be processed for rgb output, there is no
+need to support shortcuts like `hwb($color, $alpha)`. However, it is reasonable
+to support both the space-delimited CSS syntax, as well as the comma-delimited
+Sass named-argument syntax.
+
+## Semantics
+
+### `hwb()`
+
+* ```
+  hwb($hue, $whiteness, $blackness, $alpha: 1)
+  ```
+
+  * If any of `$hue`, `$whiteness`, `$blackness`, or `$alpha` aren't numbers,
+    throw an error.
+
+  * If either of `$whiteness` or `$blackness` is not a percentage between 0% and
+    100%, throw an error.
+
+  * Let `hue` be `($hue % 360) / 6`.
+
+  * Let `whiteness` and `$blackness` be the respective values of `$whiteness`
+    and `$blackness`
+
+  * If the sum of `whiteness` and `blackness` is greater than 100%, normalize
+    both to maintain the same relative ratio, adding up to 100% combined.
```

Comment:
Please describe the process of normalizing these in more detail, ideally with a formula.